app/src/main/java/com/example/tungcon/demo/movie/main/MainActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.example.tungcon.demo.movie.main;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import com.example.tungcon.demo.R;
+import com.example.tungcon.demo.movie.MovieActivity;
+
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+    private Button mClick;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        initView();
+    }
+
+    private void initView() {
+        mClick = findViewById(R.id.button_click);
```

Comment:
nếu chỉ để set click
em có thể viết 
```suggestion
        findViewById(R.id.button_click).setOnClickListener(this);
```